.gitignore

```diff
@@ -24,3 +24,5 @@ go.work
 *.swp
 
 periph_board
+exec
+module.tar.gz
```

Comment:
I've got a fairly strong preference to change your text editor/IDE to add newlines to the ends of all files. Half the text editors do this by default and half don't, at which point we get all these spurious diffs where the newline gets added and removed and added again. The files are not POSIX compliant without the newline at the end, and you get the red warning Θ when viewing the file on github as a result.

For tips, see https://thoughtbot.com/blog/no-newline-at-end-of-file I'd be happy to help you change your editor if you want.